omod/src/main/webapp/resources/scripts/custom/deletePatient.js

```diff
@@ -0,0 +1,61 @@
+var delPatient = delPatient || {};
+
+delPatient.deletePatientCreationDialog = null;
+
+/**
+ * Functions used to Delete a Patient (uses DeletePatientFragmentController)
+ */
+
+delPatient.returnUrl = "/coreapps/findpatient/findPatient.page?app=coreapps.findPatient";
+
+delPatient.createDeletePatientCreationDialog = function() {
+    delPatient.deletePatientCreationDialog = emr.setupConfirmationDialog({
+        selector: '#delete-patient-creation-dialog',
+        actions: {
+            confirm: function() {
+                var deleteReason = jQuery('#delete-reason').val().trim(); //Retrieve text from text box
+                if(deleteReason && deleteReason.length > 0) { //Should not be invalid or empty
+                    jq.ajax({
+                        url: '/openmrs/ws/rest/v1/patient/'+delPatient.patientUUID,
+                        type: 'DELETE',
+                        body: {
+                            "void_reason": deleteReason
+                        },
+                        success: function() {
+                            emr.successMessage("Success");
+                            jq('#delete-patient-creation-dialog' + ' .icon-spin').css('display', 'inline-block').parent().addClass('disabled');
+                            delPatient.deletePatientCreationDialog.close();
+                            jq(setTimeout(delPatient.goToReturnUrl, 1300)); //Allow the success message to display before redirecting to another page
```

Comment:
I string searched the entire project but could not find one instance of a JS file using a sticky toast message. If you could post the link to a file that does, then I could go ahead and change it. I mostly followed the conventions in other files...just so that everything looks coherent